specs/core/0_fork-choice.md

```diff
@@ -136,17 +136,72 @@ def get_latest_attesting_balance(store: Store, root: Root) -> Gwei:
     ))
 ```
 
+#### `filter_block_tree`
```

Comment:
missing in TOC

Reply:
Thanks! fixed the fork choice TOC in #1510 which is under review for this release